Pre-commit hooks fail: missing type stubs for Azure datalake.

**Describe the bug**
Pre-commit hooks fail, like such:
```
(chainlit-py3.10) drbob@seahorse backend % git commit -m 'Add black and isort to dev dependencies group.'
→ No staged files match any configured task.

> @ lintPython /Users/drbob/Development/chainlit
> cd backend && poetry run mypy chainlit/

chainlit/data/storage_clients.py:4: error: Skipping analyzing "azure.storage.filedatalake": module is installed, but missing library stubs or py.typed marker  [import-untyped]
chainlit/data/storage_clients.py:4: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 51 source files)
```

**To Reproduce**
Steps to reproduce the behavior:

1. Checkout branch from #1191 (so with lockfile, for consistent replication).
2. Make some arbitrary changes.
3. Call `git commit`.
4. Aforementioned error.

**Context**
OS: macOS 14.5 (Sonoma
Python: 3.10.14
mypy: 1.11.1
azure-storage-file-datalake: 12.16.0

--- a/backend/pyproject.toml
+++ b/backend/pyproject.toml
@@ -103,6 +103,7 @@ module = [
     "vertexai.language_models",
     "vertexai.preview.generative_models",
     "google.cloud",
+    "azure.storage.filedatalake",
 ]
 ignore_missing_imports = true
 

